### PR TITLE
Report: convert rest of URL list audits

### PR DESCRIPTION
**Pull request by patrickhulce**
didn't actually implement a URL list renderer since tables seem to be fine

image

### CODE REVIEW

File: lighthouse-core/audits/audit.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Audit {
     const tableRows = results.map(item => {
       return headings.map(heading => {
         const value = item[heading.key];
-        if (typeof value === 'object' && value.type) return value;
+        if (typeof value === 'object' && value && value.type) return value;
```
Comment (paulirish) [inline lighthouse-core/audits/audit.js]: the 2nd condition? how would it pass the typeof if it was already falsey?

Comment (patrickhulce) [inline lighthouse-core/audits/audit.js]: `null` 💔

Comment (paulirish) [inline lighthouse-core/audits/audit.js]: ah my bad. ;)

**Comment by ebidel** [comment]
Do you want to add a few assert in these tests that make sure `details` is there?

**Comment by patrickhulce** [comment]
> Do you want to add asserts in the tests to make sure details is there?

👍 done

**Pull request merged**